docs/about/community.md

```diff
@@ -10,19 +9,36 @@ custom_edit_url: null
 
 We welcome the participation of anyone interested in contributing to the projects in our portfolio. You can work on open issues on projects in our portfolio, create a new issue, or e-mail us if you want to be more involved.
 
+Join the conversation in the Open Source with SLU [Slack workspace](https://join.slack.com/t/oswslu/shared_invite/zt-24f0qhjbo-NkSfQ4LOg5wXxBdxP4vzfA).
+
+## Overview
+
+* Join our Slack workspace.
+* Contribute changes to an Open Source with SLU project.
+* Add your name to the Contributors list.
+* After making five contributions, you can be added to the `oss-slu` team in GitHub by contacting one of the program staff.
+
+## Getting Startted
+
```

Comment:
I think in "getting started" we need more details on how community members can start contributing to the projects. Maybe some of the details from the contributors PR template could be moved here.

Reply:
updated